docs/explanation/launchpad.rst

```diff
@@ -14,18 +14,18 @@ infrastructure.
 Launchpad features, among others, are:
 
 - **Bugs**: :term:`Bug Tracking System`
-- **Code**: :term:`source code <Source Code>` hosting with :term:`git` or :term:`Bazaar`,
+- **Code**: :term:`source code <Source Code>` hosting with :term:`Git` or :term:`Bazaar`,
   :term:`version control <Version Control System>` and :term:`code review <Code Review>` features
 - **Answers**: community support site and knowledge base
-- **Translations**: collaboration platform for localizing software 
+- **Translations**: collaboration platform for localising software
 - **Blueprints**: feature planning and specification tracking
 - Ubuntu :term:`package <Package>` building and hosting
 - Team/Group management
 
 While platforms like GitHub put users and groups at the top level, Launchpad
 puts projects at the top level. If you take Ubuntu as an example, you can
-see that you can access it at the top level: https://launchpad.net/ubuntu.
-Users and groups begin with a ``~``, for instance https://launchpad.net/~techboard.
+see that you can access it at the top level: `https://launchpad.net/ubuntu`_.
+Users and groups begin with a ``~``, for instance `https://launchpad.net/~ubuntu-foundations-team`_.
```

Comment:
question: Why did this needed to be changed?

Reply:
I just a picked a team that wouldn't be flagged by the spellcheck (like 'techboard' is) :) Given that it's an example, I believe kt doesn't matter.